common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -474,6 +474,8 @@ enum class eChannelScanOperationCode : uint8_t {
 #define CHANNEL_SCAN_INVALID_PARAM -1
 #define SCAN_ALL_CHANNELS 0
 
+#define CLIENT_PARAMETER_NOT_CONFIGURED -1
```

Comment:
```suggestion
#define PARAMETER_NOT_CONFIGURED -1
```

why limit to just client parameters - this define can be used to any parameter

Reply:
Changed